hypertrace-metrics-processor/hypertrace-metrics-processor/src/main/resources/configs/common/application.conf

```diff
@@ -0,0 +1,34 @@
+service.name = hypertrace-metrics-processor
+service.admin.port = 8099
+
+main.class = org.hypertrace.metrics.processor.MetricsProcessor
+
+input.topic = "otlp-metrics"
+output.topic = "enriched-otlp-metrics"
+
+precreate.topics = false
+precreate.topics = ${?PRE_CREATE_TOPICS}
+
+kafka.streams.config = {
+  application.id = metrics-processor-job
+  num.stream.threads = 2
+  num.stream.threads = ${?NUM_STREAM_THREADS}
+
+  bootstrap.servers = "localhost:9092"
+  bootstrap.servers = ${?KAFKA_BOOTSTRAP_SERVERS}
+
+  schema.registry.url = "http://localhost:8081"
+  schema.registry.url = ${?SCHEMA_REGISTRY_URL}
+  value.subject.name.strategy = "io.confluent.kafka.serializers.subject.TopicRecordNameStrategy"
```

Comment:
why is it TopicRecordNameStrategy ? are there going to be different message types sent on the topic ?